ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -198,9 +199,13 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
                             return (
                                 <div className='application-details'>
                                     <Page
-                                        title='Application Details'
+                                        title={`${this.props.match.params.name} / Application Details`}
+                                        useTitleOnly={true}
                                         toolbar={{
-                                            breadcrumbs: [{title: 'Applications', path: '/applications'}, {title: this.props.match.params.name}],
+                                            breadcrumbs: [
+                                                {title: 'Applications', path: '/applications'},
+                                                {title: <ApplicationsDetailsAppDropdown appName={this.props.match.params.name} />}
```

Comment:
The title is used as the key, which is supposed to be a string, for each breadcrumb path in top-bar.

Reply:
Thank you for catching it! These is a bug in `argo-ui`. Created PR that fixes it and allows using `React.Node`: https://github.com/argoproj/argo-ui/pull/176

Reply:
Hi @alexmt, that change in 176 looks ok to me.